discovery/install/install.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The Prometheus Authors
+// Copyright 2021 The Prometheus Authors
```

Comment:
We do not change the copyright of existing files.

Reply:
Ack, I'll revert the copyright updates on existing files